bootstrap

```diff
@@ -4,6 +4,13 @@
 # Any flags will be passed on to CMake, e.g.,
 # ./bootstrap -DCMAKE_BUILD_TYPE="Debug"
 
+## Check to make cmake is installed, if not offer to install if apt is
+## available or exit
+if ! command -v cmake >/dev/null 2>&1; then
```

Comment:
I wasn't able to fully investigate but this doesn't seem to work correctly on macos, it seems that `!` does not negate but manipulates does something with the command history.

Reply:
Really? Works on my machine. Are you using a different shell?